tests/Agent/UnitTests/Core.UnitTest/Configuration/DefaultConfigurationTests.cs

```diff
@@ -1006,6 +1005,7 @@ public string IgnoreErrorsAndIgnoreClassesCombineTests(string[] ignoreClasses, s
             return string.Join(",", _defaultConfig.IgnoreErrorsConfiguration.Keys);
         }
 
+#pragma warning disable NUnit1005 // The type of the value specified via ExpectedResult must match the return type of the method
```

Comment:
Should we submit a tech debt ticket so that we don't need this pragma anymore?

Reply:
I'll just refactor real quick. Not a big deal.